Index: controller/addScoreController.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import User from '../models/User';\n\nconst HTTP_STATUS = {\n    NotFound: 404,\n    OK: 200,\n    InternalServerError: 500\n};\n\nconst MESSAGES = {\n    UserNotFound: 'User not found'\n};\n\nexport const addScore = async (req, res) => {\n    const { userId } = req.params;\n    const { score } = req.body;\n\n    try{\n        const user = await User.findById(userId);\n        if(!user) return res.status(HTTP_STATUS.NotFound, MESSAGES.UserNotFound);\n        await user.addScore(score);\n        res.status(HTTP_STATUS.OK).json({\n            success: true,\n            message: 'Score added successfully',\n            score: user.scores\n        });\n    } catch (error) {\n        res.status(HTTP_STATUS.InternalServerError).json({\n            message: 'Failed to add score'\n        });\n    }\n};\n
===================================================================
diff --git a/controller/addScoreController.js b/controller/addScoreController.js
--- a/controller/addScoreController.js	(revision f2cfab45254b43712f832ce95d3b7910cd0c9675)
+++ b/controller/addScoreController.js	(date 1752222526697)
@@ -1,5 +1,3 @@
-import User from '../models/User';
-
 const HTTP_STATUS = {
     NotFound: 404,
     OK: 200,
Index: routes/userRoutes.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import express from 'express';\nimport {addScore} from \"../controller/addScoreController.js\";\n\nconst router = express.Router();\n\nrouter.post('/score/:id',addScore);\n\nexport default router;
===================================================================
diff --git a/routes/userRoutes.js b/routes/userRoutes.js
--- a/routes/userRoutes.js	(revision f2cfab45254b43712f832ce95d3b7910cd0c9675)
+++ b/routes/userRoutes.js	(date 1752187418192)
@@ -1,8 +1,8 @@
 import express from 'express';
-import {addScore} from "../controller/addScoreController.js";
+import { addScore } from "../controller/addScoreController.js";
 
-const router = express.Router();
+const userRouter = express.Router();
 
-router.post('/score/:id',addScore);
+userRouter.post('/score/:id', addScore);
 
-export default router;
\ No newline at end of file
+export default userRouter;
\ No newline at end of file
